<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="MyView" name="myView">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::FullViewportUpdate</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QQuickWidget" name="quickWidget">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>331</height>
       </size>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::SizeRootObjectToView</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChange_Plane_Settings">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAddNewZone"/>
    <addaction name="actionChangeExistingZone"/>
    <addaction name="actionDeleteZone"/>
    <addaction name="actionSet_Airport"/>
    <addaction name="separator"/>
    <addaction name="actionCompute"/>
    <addaction name="actionClear_Screen"/>
    <addaction name="separator"/>
    <addaction name="actionChangePlaneSettings"/>
    <addaction name="actionStartSimFlight"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChange_Plane_Settings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddNewZone">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add new zone</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionChangePlaneSettings">
   <property name="text">
    <string>Change plane settings...</string>
   </property>
  </action>
  <action name="actionCompute">
   <property name="text">
    <string>Compute</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionChangeExistingZone">
   <property name="text">
    <string>Change existing zone</string>
   </property>
  </action>
  <action name="actionDeleteZone">
   <property name="text">
    <string>Delete Zone</string>
   </property>
  </action>
  <action name="actionSet_Airport">
   <property name="text">
    <string>Set Airport</string>
   </property>
  </action>
  <action name="actionClear_Screen">
   <property name="text">
    <string>Clear Screen</string>
   </property>
  </action>
  <action name="actionStartSimFlight">
   <property name="text">
    <string>Start Sim Flight</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>MyView</class>
   <extends>QGraphicsView</extends>
   <header>myview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
